### PR TITLE
Many improvements to the documentation

### PR DESCRIPTION
**Pull request by JerBouma**
These things:

- Extended FAQ for Terminal and SDK
- Added Troubleshooting page to Terminal FAQ
- Integrated #3539
- Made SDK basics more coherent with Terminal
- Many grammar changes and smaller things

### CODE REVIEW

**Review by jose-donato — APPROVED** [review]
Lgtm

**Comment by JerBouma** [comment]
Do not merge just yet, need to check some grammar first tomorrow.

File: website/content/sdk/guides/intros/portfolio/po.md [hunk 1]
```diff
@@ -4,18 +4,10 @@ keywords: ["portfolio", "optimization", "mean variance", "risk parity", "black l
...
-By providing Excel templates, the user can make sense of the vast array of parameters that each command has. E.g. think
-of the historic period you wish to use or which of the more than 10 risk measures and covariance methods should be used?
-These are questions the templates make easier to answer.
+The Portfolio Optimization menu allows the user to apply advanced optimization techniques to a portfolio of any type and of any size. It does so by introducing a multitude of optimization techniques ranging from <a href="https://www.investopedia.com/terms/m/meanvariance-analysis.asp" target="_blank">mean-variance optimization</a> to <a href="https://www.investopedia.com/terms/r/risk-parity.asp" target="_blank">risk parity models</a> and <a href="https://www.investopedia.com/terms/c/cluster_analysis.asp" target="_blank">hierarchical clustering models</a>. By providing Excel templates, the user can make sense of the vast array of parameters that each command has. E.g. think of the historic period you wish to use or which of the more than 10 risk measures and covariance methods should be used? These are questions the templates make easier to answer.
```
Comment (jose-donato) [inline website/content/sdk/guides/intros/portfolio/po.md:8]: always add `rel="noreferrer noopener"` when doing `target="_blank"`

File: website/content/sdk/guides/intros/portfolio/po.md [hunk 2]
```diff
@@ -4,18 +4,10 @@ keywords: ["portfolio", "optimization", "mean variance", "risk parity", "black l
...
+The capabilities of the [Portfolio Optiminzation menu](/terminal/guides/portfolio/po) from the OpenBB Terminal are wrapped into a powerful SDK, enabling users to work with the data in a flexible environment that can be fully customized to meet the needs of any user. These functionalities allow the user to apply advanced optimization techniques to a portfolio of any type and of any size. It does so by introducing a multitude of optimization techniques ranging from <a href="https://www.investopedia.com/terms/m/meanvariance-analysis.asp" target="_blank">mean-variance optimization</a> to <a href="https://www.investopedia.com/terms/r/risk-parity.asp" target="_blank">risk parity models</a> and <a href="https://www.investopedia.com terms/c/cluster_analysis.asp" target="_blank">hierarchical clustering models</a>. By providing Excel templates, the user can make sense of the vast array of parameters that each command has. E.g. think of the historic period you wish to use or which of the more than 10 risk measures and covariance methods should be used? These are questions the templates make easier to answer.
```
Comment (jose-donato) [inline website/content/sdk/guides/intros/portfolio/po.md:10]: same as above

File: website/content/sdk/guides/intros/portfolio/po.md [hunk 3]
```diff
@@ -24,30 +16,23 @@ Start a Python script or Notebook file by importing the module:
...
 
-The OpenBB Terminal does, however, allow the user to run any model despite the `technique` you selected. Therefore, if
-you are interested in running multiple models, consider removing the value for `technique`. Do note that this makes it
-more difficult to understand which values are used for which model.
+The OpenBB Terminal does, however, allow the user to run any model despite the `technique` you selected. Therefore, if you are interested in running multiple models, consider removing the value for `technique`. Do note that this makes it more difficult to understand which values are used for which model.
 
 <a target="_blank" href="https://user-images.githubusercontent.com/46355364/171144692-dd812efd-1e95-4a71-a93f-7ae8a480fe5d.png"><img alt="OpenBB Parameters Template" src="https://user-images.githubusercontent.com/46355364/171144692-dd812efd-1e95-4a71-a93f-7ae8a480fe5d.png"></img></a>
```
Comment (jose-donato) [inline website/content/sdk/guides/intros/portfolio/po.md:29]: same as above

File: website/content/sdk/guides/intros/portfolio/po.md [hunk 4]
```diff
@@ -118,10 +97,7 @@ Which returns:
 | TLT   |       0.21927 |
 | TSM   |       0.04626 |
```
Comment (jose-donato) [inline website/content/sdk/guides/intros/portfolio/po.md:99]: same as above

File: website/content/sdk/guides/intros/portfolio/po.md [hunk 5]
```diff
@@ -145,14 +121,13 @@ Which returns:
 
 
 ## Examples
-To demonstrate the capabilities of the Portfolio Optimization menu, the entire <a href="https://www.investopedia.com/terms/s/sp500.asp" target="_blank">S&P 500 index</a> (as of 30th of May 2022)
-is used and optimized and analysed in a variety of ways. Starting by loading in the dataset with the following:
+To demonstrate the capabilities of the Portfolio Optimization menu, the entire <a href="https://www.investopedia.com/terms/s/sp500.asp" target="_blank">S&P 500 index</a> (as of 30th of May 2022) is used and optimized and analysed in a variety of ways. Starting by loading in the dataset with the following:
```
Comment (jose-donato) [inline website/content/sdk/guides/intros/portfolio/po.md:124]: same as above

File: website/content/sdk/guides/intros/portfolio/po.md [hunk 6]
```diff
@@ -196,8 +171,7 @@ This results in the following (the result is edited, as it would show 500 ticker
 | ZION  |                    0.0014  |
 | ZTS   |                    0.00225 |
 
-It is possible to delve further into these findings with the `plot` functionality for example done by looking at the portfolio's returns <a href="https://www.investopedia.com/terms/h/histogram.asp" target="_blank">histogram</a>
-which also includes a variety of risk measures as well as the portfolio's drawdowns.
+It is possible to delve further into these findings with the `plot` functionality for example done by looking at the portfolio's returns <a href="https://www.investopedia.com/terms/h/histogram.asp" target="_blank">histogram</a> which also includes a variety of risk measures as well as the portfolio's drawdowns.
```
Comment (jose-donato) [inline website/content/sdk/guides/intros/portfolio/po.md:174]: same as above

**Review by jose-donato — CHANGES_REQUESTED** [review]

**Review by jose-donato — APPROVED** [review]

**Pull request merged**